pr_testing/cmssw-fake-package.spec

```diff
@@ -0,0 +1,10 @@
+### RPM cms cmssw-fake-package 1.0
+## NOCOMPILER
+## NO_VERSION_SUFFIX
+
+@provides@
```

Comment:
why not just have one spec and just add the `@provides@` in cmssw-pr-package.spec? It could be replaced with empty string if non used otherwise we replace it withthe actual provides.